docs/proposals/simulation/simulation.md

```diff
@@ -0,0 +1,167 @@
+
+This proposal provides features in datasets, edge-cloud collaborative AI system simulation, etc. for the current version of Ianvs
+
+## Goals
+
+The project is committed to building an edge-cloud collaborative AI Bechmark for edge-cloud collaborative lifelong detection in the Kubeedge open source community.
+
+- Provide downloadable open source datasets and data processing scripts for the difficulty in obtaining datasets and supporting algorithms
+- Provides industrial-grade distributed collaborative system simulation for the problem that the repeated deployment of edge-cloud collaboration is too heavy
+
+## Proposal
+
+### Edge-Cloud Collaborative AI Dataset and Corresponding Data Processing Script
+
+It will provide an open-source edge-cloud collaborative AI dataset management module that is convenient for algorithm developers to use, which can be used as a plug-in for Sedna. The following functions will be supported:
+
+- Dataset management of edge-cloud collaborative AI
+- Quick download and use of datasets
+
+### Industrial Distributed Collaborative System Simulation
+
+It will provide Ianvs with docker in docker system emulation features.
+
+## Design Details
+
+### Dataset automatic download function
+
+The current `testenv.yaml` file is as follows
+
+```yaml
+testenv:
+  # dataset configuration
+  dataset:
+    # the url address of train dataset index; string type;
+    train_url: "/ianvs/dataset/train_data/index.txt"
+    # the url address of test dataset index; string type;
+    test_url: "/ianvs/dataset/test_data/index.txt"
+
+  # metrics configuration for test case's evaluation; list type;
+  metrics:
+      # metric name; string type;
+    - name: "f1_score"
+      # the url address of python file
+      url: "./examples/pcb-aoi/singletask_learning_bench/testenv/f1_score.py"
+```
+
+For the convenience of developers, the expected testenv.yaml file is as follows:
+
+- The user fills in the `dataset_url`, e.g. HUAWEI CLOUD OBS link.
+- The user fills in `dataset_path`, where the dataset wants to be saved.
+
+```yaml
+testenv:
+  # dataset configuration
+  dataset:
+    # the url address of dataset with zip file; string type;
+    dataset_url: "https://kubeedge.obs.cn-north-1.myhuaweicloud.com:443/ianvs/pcb-aoi/dataset.zip"
+    # the url address of saving dataset
+    dataset_path: "./dataset/"
+
+  # metrics configuration for test case's evaluation; list type;
+  metrics:
+      # metric name; string type;
+    - name: "f1_score"
+      # the url address of python file
+      url: "./examples/pcb-aoi/singletask_learning_bench/testenv/f1_score.py"
+```
+
+The expected implementation method is to modify `dataset.py` in the `testenvmanager` module as follows:
+
+1. Add `self.dataset_url`, `self.dataset_path`
+2. Add `download_save()` function
+
+![](https://github.com/iszhyang/images/blob/main/ianvs/dataset-download.png?raw=true)
```

Comment:
It is suggest to put the image file to this repository, not a link to your private repository.